File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums)
        ans = []
        for i in range(length):
            ans.append(nums[i])
        for i in range(length):
            ans.append(nums[i])
        

        return ans
            

        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        Array.Sort(nums);
        int i = 0;
        while( i<nums.Length-1){
            if(nums[i] == nums[i+1])
            {
                return true;
            }
            else i++;
        }

        return false;

    }
}